src/diffcalc_API/server.py

```diff
@@ -31,6 +32,9 @@
 
 @app.exception_handler(DiffcalcException)
 async def diffcalc_exception_handler(request: Request, exc: DiffcalcException):
+    tb = traceback.format_exc()
+    logging.warn(f"Diffcalc Exception caught by middleware: {tb}")
```

Comment:
If you do this then it will use the logger you made above and then it will produce `__name__` in the output (it might do this automatically anyway, not sure). 

```suggestion
    logger.warn(f"Diffcalc Exception caught by middleware: {tb}")
```

Reply:
whoops, that was a typo. Changing now.